cfme/tests/configure/test_visual_infra.py

```diff
@@ -360,8 +357,7 @@ def test_vm_noquads(appliance, request, set_vm_quad):
     """
     view = navigate_to(appliance.collections.infra_vms, 'VMsOnly')
     view.toolbar.view_selector.select('Grid View')
-    # Here data property will return an empty dict when the Quadrants option is deactivated.
-    assert not view.entities.get_first_entity().data
+    assert 'topRight' not in view.entities.get_first_entity().data.get('quad', {})
```

Comment:
view.entities.get_first_entity().data.get('quad', {}) will anyway return empty dict, using quadicon instead of quad returns dictionary with some data.

Reply:
where do you use quadicon ? I don't see it in data returned by js.